yarn.lock

```diff
@@ -27010,7 +27010,7 @@ y18n@^3.2.1:
   integrity sha512-uGZHXkHnhF0XeeAPgnKfPv1bgKAYyVvmNL1xlKsPYZPaIHxGti2hHqvOCQv71XMsLxu1QjergkqogUnms5D3YQ==
 
 y18n@^4.0.0:
-  version "4.0.1"
+  version "4.0.0"
```

Comment:
is this change necessary ?

Reply:
I didn't make that change, should I revert it?

Reply:
yes, it's downgrading a patch version... so unless there's a reason to do so I'd rather not

Reply:
so actually there is an inconsistency in our yarn.lock 

https://github.com/microsoft/fluentui/pull/22748